system/Exceptions/CastException.php

```diff
@@ -43,20 +43,4 @@ public static function forInvalidJsonFormatException(int $error)
 				return new static(lang('Cast.jsonErrorUnknown'));
 		}
 	}
```

Comment:
Shouldn't this exception be gone in favor of the other?

Reply:
This would be BC, this exception were added before version 4.1

Reply:
If I'm not mistaken, system exceptions are strictly for internal use and is not covered by BC. Wait, I'll check.

Reply:
OK. That was for language strings. Do add `@deprecated` tag for the class instead.